index.html

```diff
@@ -15,6 +15,98 @@
+            </article>
+            <article class="col-4">
+                <img class="col-12" src="https://wallup.net/wp-content/uploads/2016/02/206515-walls-graffiti-colorful-Miami-nature.jpg"  />
+            </article>
+            <article class="col-4">
+                <img class="col-12" src="https://i.ytimg.com/vi/2HXb9aFLm-4/maxresdefault.jpg" />
+            </article>
+            <article class="col-4">
+                <img class="col-12" src="https://images.unsplash.com/photo-1533142266415-ac591a4deae9?ixlib=rb-1.2.1&auto=format&fit=crop&w=1000&q=80"  />
+            </article>
+            <article class="col-4">
+                <img class="col-12" src="https://media.gettyimages.com/photos/man-in-a-green-shirt-walking-past-a-colorful-graffiti-wall-bowery-new-picture-id550448645?s=594x594"  />
+            </article>
+            <article class="col-4">
+                <img class="col-12" src="https://i.pinimg.com/originals/32/fa/7f/32fa7faf3f07b52994379a617b25c146.jpg"  />
+            </article>
+            <article class="col-4">
+                <img class="col-12" src="https://d15z338h2xlr9d.cloudfront.net/wp-content/uploads/2017/03/Amazing-Graffiti-and-Street-Art-Tours.jpg"  />
+            </article>
+            <article class="col-4">
+                <img class="col-12" src="http://s2.favim.com/orig/150529/art-beautiful-colourful-cool-Favim.com-2774903.jpg" />
+            </article>
+            <article class="col-4">
+                <img class="col-12" src="http://www.quiet-corner.com/wp-content/uploads/2016/04/Abandoned-Church-Transformed-With-Colorful-Graffiti-4.jpg"  />
+            </article>
+
+
+        </div>
+    </section>
+    
+    <section>
+      <h2 class="about">About</h2>
+      <p>Lorem ipsum lorem ipsum Lorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsumLorem ipsum lorem ipsum</p>
+      
+    </section>
+    
+  <section class="container">
+    <h2 class="Contact">Contact</h2>
+    <form class="form">
+      <label class="label" for="fname">Enter your name:</label><br>
+      <input class= "input" type="text" id="fname" name="fname"><br>
+      <label class="label" for="email">Enter your e-mail:</label><br>
+      <input class= "input" type="text" id="email" name="email"><br>
+      <label class="label" for="phone">Enter your phone:</label><br>
+      <input class= "input" type="text" id="phone" name="phone"><br>
+      <label class="label" for="message">Enter your message here:</label><br>
+      <input class= "input" type="text" id="message" name="message"><br>
+      <input class= "input" type="submit" value="Submit">
+    </form>
```

Comment:
Good form structure 👍 
Can you style your inputs to make them look like the design? 🙂